net-mail/mu/mu-1.2.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools elisp-common
+
+DESCRIPTION="Set of tools to deal with Maildirs, in particular, searching and indexing"
+HOMEPAGE="http://www.djcbsoftware.nl/code/mu/"
+SRC_URI="https://github.com/djcb/mu/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="guile emacs zsh-completion"
+
+# net-mail/mailutils also installes /usr/bin/mu.  Block it until somebody
+# really wants both installed at the same time.
+DEPEND="
+	!net-mail/mailutils
+	dev-libs/xapian
```

Comment:
x comes after g, but mu needs at least 1.4.

(Version requirement found in https://github.com/djcb/mu/blob/1.2/configure.ac)